src/Trips.js

```diff
@@ -11,29 +11,16 @@ class Trips {
     });
   }
 
-  // retrieveTripDestinationsForUser(id) {
-  //   const destinations = new Destinations()
-  //   const tripDestinationIds = this.retrieveTripsForUser(id).map((trip) => {
-  //     return trip.destinationID;
-  //   })
-  //   console.log(destinations)
-  //   console.log(tripDestinationIds)
-  //   // const tripDestinationNames = destinations.filter(())
-  // }
-}
-
-
-
-// {
-// "id": 1,
-// "userID": 44,
-// "destinationID": 49,
-// "travelers": 1,
-// "date": "2022/09/16",
-// "duration": 8,
-// "status": "approved",
-// "suggestedActivities": []
-// },
+  retrieveTripCost(destinations, trip) {
```

Comment:
I like how you bring in destinations. I don't think i would have thought to do it that way. Neat!